src/WebJobs.Script/StorageProvider/IAzureStorageProvider.cs

```diff
@@ -11,6 +12,8 @@ namespace Microsoft.Azure.WebJobs.Script
     /// </summary>
     public interface IAzureStorageProvider
     {
+        IConfiguration Configuration { get; }
```

Comment:
See comment about the property.

Reply:
Removed this property and added ConnectionExists() method to AzureStorageProvider.